mltsp/celery_tasks.py

```diff
@@ -1,15 +1,24 @@
 from celery import Celery
 import os
 import sys
+import yaml
 sys.path.append(os.path.dirname(os.path.dirname(__file__)))
-from mltsp.cfg import config
 from mltsp import time_series
 from mltsp import util
 from mltsp import featurize_tools as ft
 
 
-celery_app = Celery('celery_fit', broker=config['celery']['celery_broker'])
-celery_app.config_from_object(config['celery'])
+celery_config = {
+    'CELERY_ACCEPT_CONTENT': ['pickle'],
+    'CELERY_IMPORTS': ['mltsp', 'celery_tasks'],
+    'CELERY_RESULT_BACKEND': 'amqp',
+    'CELERY_RESULT_SERIALIZER': 'pickle',
+    'CELERY_TASK_SERIALIZER': 'pickle',
+    'INSTALLED_APPS': ['mltsp'],
+    'CELERY_BROKER': 'amqp://guest@localhost//'
+}
+celery_app = Celery('celery_fit', broker=celery_config['CELERY_BROKER'])
+celery_app.config_from_object(celery_config)
```

Comment:
Oh right, we talked about that yesterday; it's just hard-coded now since if you tried to change any of it you would probably break everything...updated the PR text to reflect this.